### PR TITLE
try ginkgo directly

### PR DESCRIPTION
**Pull request by MHBauer**
- [ ] document your feature
 - [ ] comment your code
 - [ ] update the README / how-to-run
 - [ ] have tests
 - [ ] get reviews

we need to be able to filter the tests so we can run them locally without vs on the server.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ build:
 	go build -v github.com/cloudfoundry-incubator/blockhead/cmd/broker
 
 test:
-	go test -v ./...
+	ginkgo -v -p -r -race -randomizeAllSpecs
```
Comment (nimakaviani) [inline Makefile]: I like!

**Comment by nimakaviani** [comment]
LGTM

File: .travis.yml [hunk 2]
```diff
@@ -5,6 +5,10 @@ language: go
 services:
   - docker
 
+script:
+  - go install github.com/onsi/ginkgo/ginkgo
```
Comment (nimakaviani) [inline .travis.yml]: from their github page:

```
go get -u github.com/onsi/ginkgo/ginkgo  # installs the ginkgo CLI
go get -u github.com/onsi/gomega/...     # fetches the matcher library
```

Comment (MHBauer) [reply .travis.yml]: that's exactly what we want to avoid. we already have the code checked in.

**Comment by MHBauer** [comment]
@swetharepakula I think this does the right thing now.

**Comment by nimakaviani** [comment]
pulling it in. looks like it works.

**Pull request merged**